build.rs

```diff
@@ -299,12 +299,41 @@ fn build_translator(compiled_libraries: &mut HashSet<Libs>, target: &String) {
             .flag_if_supported("-arch:SSE2"); // MSVC
     }
 
-    build.link_lib_modifier("-whole-archive");
```

Comment:
Why is this removed?

I think this is/was needed for external uses of dlls. I can recheck this on my windows machine later today.

Reply:
hmm, I could be wrong. I didn't test on windows machine. Please let me know if it fails on windows machine, I will add it back. Same for the next comment!

Reply:
Works for me in wr.